improver_tests/utilities/test_mathematical_operations.py

```diff
@@ -494,5 +494,48 @@ def test_dimension_preservation(self):
         self.assertListEqual(result_coord_order, expected_coord_order)
 
 
+class Test_fast_linear_fit(IrisTest):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Class name "Test_fast_linear_fit" doesn't conform to '[A-Z_][a-zA-Z0-9]+$' pattern](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5319684)